### PR TITLE
Various changes (mainly NSBundle) to make it easier to build an embeddable library/framework

### PR DESCRIPTION
**Pull request by lemonmojo**
- Replace `[NSImage imageNamed:]` with `[NSImage it_imageNamed:forClass:]`
- Replace `[NSBundle mainBundle]` with `[NSBundle bundleForClass:]`
- Pass `[NSBundle bundleForClass:]` to `initWithNibName:bundle:` calls
- Expose some `PTYTextView` properties and methods
  - `showTimestamps`
  - `anyAnnotationsAreVisible`
  - `selectCurrentCommand:`
  - `selectOutputOfLastCommand:`
- In `PTYSession`, handle `paste:` if not sent from an `NSMenuItem`

### CODE REVIEW

File: sources/PTYTextView.h [hunk 1]
```diff
@@ -331,6 +331,10 @@ typedef NS_ENUM(NSInteger, PTYTextViewSelectionExtensionUnit) {
 typedef void (^PTYTextViewDrawingHookBlock)(iTermTextDrawingHelper *);
 @property(nonatomic, copy) PTYTextViewDrawingHookBlock drawingHook;
 
+@property(nonatomic, assign) BOOL showTimestamps;
```
Comment (gnachman) [inline sources/PTYTextView.h]: Why was it necessary to expose these properties?

File: sources/PTYTextView.h [hunk 2]
```diff
@@ -589,6 +593,9 @@ typedef void (^PTYTextViewDrawingHookBlock)(iTermTextDrawingHelper *);
 
 - (void)setTransparencyAffectsOnlyDefaultBackgroundColor:(BOOL)value;
 
+- (IBAction)selectCurrentCommand:(id)sender;
```
Comment (gnachman) [inline sources/PTYTextView.h]: Most IBActions aren't exposed in the header file. IB is able to find them in .m files. Are these needed for a particular purpose?

File: sources/PTYTextView.h (hunk 1 above)
Comment (lemonmojo) [reply sources/PTYTextView.h]: I'm using these in my embeddable iTerm2 control to change the title/state of menu items.
For the `showTimestamps` case I'm showing `Hide Timestamps` if it's `YES` and `Show Timestamps` if it's `NO`.

File: sources/PTYTextView.h (hunk 2 above)
Comment (lemonmojo) [reply sources/PTYTextView.h]: Of course, if you wire up those actions using IB, they can stay in the implementation file. If however you need to access those methods from another class, they need to be visible in the header. That's why I've exposed them.

**Comment by lemonmojo** [comment]
@gnachman Just added exactly that.

**Comment by gnachman** [comment]
Merged as ec5d3269c21a6327db7bed35e15db72c74150175. Thanks!

**Pull request closed without merging**